gensim/models/doc2vec_inner.pyx

```diff
@@ -2,33 +2,27 @@
 # cython: boundscheck=False
 # cython: wraparound=False
 # cython: cdivision=True
+# cython: embedsignature=True
 # coding: utf-8
 #
 # Copyright (C) 2013 Radim Rehurek <me@radimrehurek.com>
 # Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
-
+"""Optimized cython functions for training :class:`~gensim.models.doc2vec.Doc2Vec` model."""
```

Comment:
Missing blank line after docstring (here and elsewhere).

Reply:
No needed, see https://github.com/numpy/numpy/blob/master/numpy/ma/mrecords.py#L10

Reply:
Strong -1: goes against standard Python docstring logic (blank line to delimit unrelated blocks). Also, ugly / harder to read for no good reason (or is there a reason?).

Reply:
> goes against standard Python docstring logic

any PEP for it? Also see https://www.python.org/dev/peps/pep-0257/#one-line-docstrings